xapian-core/include/xapian/cluster.h

```diff
@@ -219,8 +219,8 @@ class Centroid : public PointType {
     /// Constructor
     Centroid() { magnitude = 0; }
 
-    /// This method initializes the values of a centroid to the Point 'x'
-    void set_to_point(Point &x);
+    // Constructor with Point argument
+    Centroid(Point &x);
```

Comment:
Do you think this should be an `explicit` constructor? It feels like having `Centroid& c = p;` work might be unexpected.